Yank/tests/test_yank.py

```diff
@@ -31,6 +32,7 @@
 from yank.repex import ThermodynamicState
 from yank.pipeline import find_components
 
+
 from yank.yank import *
```

Comment:
I can work to fix that, I think its the only one and its been in since before I started working on the code. I wanted to make sure the changes I made for functionality changes worked before I did code cleanup.

If we want a full code cleanup, pycharm is more than happy to tell me all our PEP 8 violations 😈